requirements.txt

```diff
@@ -1,12 +1,11 @@
 ## data
-numpy~=1.20.3
-pillow~=8.2.0
-pystac[validation]==0.5.6
-python-dateutil~=2.8.1
-requests~=2.25.1
-sentinelsat>=1.0.1
-shapely~=1.7.1
-pydantic
+numpy
+pillow
+pystac[validation]==1.0.0
+python-dateutil
+requests
+sentinelsat>=0.14

```

Comment:
I guess something went wrong when you resolved the conflict, can you update the version again? `pydantic` is also missing.

Reply:
ok , i will do that